web-console/knikubevirt/VMIs/VMIs.md

```diff
@@ -0,0 +1,66 @@
+# Surfacing VMIs in the UI
+
+VMI is the current running VM.
+The initial reason for surfacing VMIs in the UI was that there is no point in hiding the VMIs from the UI.
+
+## UCs
+
+1. Debugging - If a user thinks that something isn’t right with the VMs and he wants to check what’s going on. When a VMI isn’t managed by a VM and it’s invisible in the UI but that VMI is actually still there because it’s impossible to delete VMIs (that’s a basic Kubernetes mechanism). This can happen by an orphan delete, or that a replica set isn’t used.
+
+2. On Bare metal machines installed with Kubernetes or users who install Kubevirt for CNV - On these VMs there are clusters that can be scaled up and down and they create the VMIs (and not the VMs).
+
+## 2 Types of personas
```

Comment:
We haven't included personas in the repo in the past have we? Maybe we shouldn't here. WDYT?